README.md

```diff
@@ -17,6 +17,7 @@ The Snowflake Ingest Service SDK depends on the following libraries:
 
 * snowflake-jdbc (3.13.30 to 3.13.33) 
 * slf4j-api
+* com.github.luben:zstd-jni (1.5.0-1)
```

Comment:
Why we decide to treat this as one of the special dependencies? @sfc-gh-lsembera has more context here if you need more help

Reply:
Hi, the problem is that zstd-jni can't be shaded. Because of that we can't include it in our sdk's shaded jar. It has to be provided separately when customers want to build their application with our sdk. I thought this was reason enough to list it with our dependencies.

Reply:
@sfc-gh-lsembera Any idea if we can do better here? Or any potential issue? I'm concerned that we keep adding to this list

Reply:
I think it is good if shade fewer dependencies, and this change goes in that direction. Another advantage of the unshaded approach here is that if the customer does not want to use zstd and wants smaller distribution, they can exclude the zstd jar from their maven project.

Reply:
Looks like we stop relocating `com.github.luben`, is there any concern that this is a behavior change?